includes/actions/class-amp-paired-post-actions.php

```diff
@@ -104,4 +105,10 @@ public static  function add_analytics_data( $amp_template ) {
 			echo AMP_HTML_Utils::build_tag( 'amp-analytics', $amp_analytics_attr, $script_element );
 		}
 	}
+
+	public static function add_generatedby_metadata( $amp_template ) {
```

Comment:
Fix WPCS error (`Squiz.Commenting.FunctionComment.Missing`). Here is a draft:

```
/**
 * Add AMP generator metadata.
 *
 * @param object $amp_template AMP_Post_Template object.
 * @since 0.6
 */
```